.gitignore

```diff
@@ -89,6 +89,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+.vscode/
```

Comment:
Note that you can put this also to your personal global git ignore, so that it would apply on all your git projects.